test/integration/targets/win_firewall_rule/tasks/main.yml

```diff
@@ -3,98 +3,94 @@
     name: http
     state: absent
     action: "{{ item }}"
-    direction: In
+    direction: in
   with_items:
      - allow
      - block
 
 - name: Add firewall rule
   win_firewall_rule:
     name: http
-    enable: yes
+    enabled: yes
     state: present
     localport: 80
     action: allow
-    direction: In
-    protocol: TCP
+    direction: in
+    protocol: tcp
   register: add_firewall_rule
 
 - name: Check that creating new firewall rule succeeds with a change
   assert:
     that:
-    - add_firewall_rule.failed == false
```

Comment:
The check does not buy us anything, if it would have failed we would not be doing the assert. (Unless we used `ignore_errors: yes`).